src/js/state/Tab/activeTabSelect.js

```diff
@@ -5,6 +5,12 @@ import type {State} from "../types"
 import type {TabState} from "./types"
 import Tabs from "../Tabs"
 
-export default function activeTabSelect<T>(fn: (TabState) => T): (State) => T {
-  return createSelector<State, void, T, TabState>(Tabs.getActiveTab, fn)
+export default function activeTabSelect<T>(
+  fn: (TabState, State) => T
+): (State) => T {
+  return createSelector<State, void, T, TabState, State>(
+    Tabs.getActiveTab,
+    (state) => state,
```

Comment:
We now pass in all of the state as the second parameter to the activeTabSelect function.